### ISSUE TITLE
Use clearer header field name or value

### ISSUE THREAD

**Issue by brodrigu** [globalprivacycontrol/gpc-spec#27]
As designed, I had to review the spec to understand that`Sec-GPC: 1` means "do not sell or share". 

The `Sec-GPC` would be more human readable if its name or value suggested the users specific preference.
 
Two quick suggestions:
`Sec-GPC: no-sell-share`
`Sec-GPC-No-Sell-Share: 1`

**Comment by SebastianZimmeck** [comment]
I agree, @brodrigu, readability is important. Though, my preference would be to leave the values as they are. For binary values `1` usually means `on`. Also, other standards have similar notation, especially, [DNT = 1](https://www.w3.org/TR/tracking-dnt/#dnt-header-field) is pretty similar to our use case.
